commands/client-pause.md

```diff
@@ -38,3 +38,7 @@ to be static not just from the point of view of clients not being able to write,
 @return
 
 @simple-string-reply: The command returns OK or an error if the timeout is invalid.
+
+## Behavior change history
```

Comment:
i'm not certain regarding the MD header. i see in the past it was done with `@history`
a395fc70a6d0fc1744070289efd6c92a44befe28
@itamarhaber PTAL